packages/realm/package.json

```diff
@@ -141,7 +154,10 @@
     },
     "build:node": {
       "dependencies": [
-        { "script": "bindgen:build:node", "cascade": false }
+        {
+          "script": "bindgen:build:node",
+          "cascade": false
+        }
       ]
     },
```

Comment:
If it's not too big of a deal, it would be great if this could be saved without formatting 👍